modules/ocf_backups/files/upload-to-box

```diff
@@ -107,30 +109,43 @@ def get_access_token(creds):
 
 def upload_to_box(creds, quiet):
     """Uploads the archive to box at the configured location"""
-    lftp_cmd = '''
-        set ftps:initial-prot ""
-        set ftp:ssl-force true
-        set ftp:ssl-protect-data true
-        open {box_ftp}
-        user {email} {passwd}
-        mirror -R --no-overwrite --parallel={concurr} {archive} {folder}
-    '''.format(
-        box_ftp=BOX_FTP,
+
+    # curl takes credentials with a "netrc file"
+    netrc_creds = 'machine ftp.box.com login {email} password {passwd}'.format(
         email=creds['email'],
         passwd=creds['password'],
-        concurr=MAX_CONCURR_UPLOADS,
-        archive=ARCHIVE,
-        folder=FOLDER,
     )
+    netrc_file = NamedTemporaryFile(mode='w')
+    netrc_file.write(netrc_creds)
+    netrc_file.flush()
```

Comment:
This should get automatically deleted when the program ends, but I think putting a comment about it getting automatically deleted on cleanup would be good since I'm not sure how many people know how `NamedTemporaryFile` works. Also, making it into a context manager so it's more explicit when it's used and when it's closed would probably be good, and then it can just be put around the one `curl` that needs it:

```
# This temporary file will get deleted when it is closed (when this context ends),
# keeping the credentials from sticking around in a more permanent file
with NamedTemporaryFile(mode='w') as netrc_file:
    netrc_file.write(netrc_creds)
    netrc_file.flush()

    # curl stuff
```

Reply:
Perhaps explicitly passing `delete=True` here would be a bit better than writing a comment.